paywall/src/data-iframe/Mailbox.ts

```diff
@@ -359,11 +364,20 @@ export default class Mailbox {
     if (typeof data !== 'object') return nukeAndReturn()
     const chainData = data as BlockchainData
     const keys = Object.keys(chainData)
-    if (keys.length !== Object.keys(this.defaultBlockchainData).length)
+    if (keys.length !== Object.keys(this.defaultBlockchainData).length) {
       return nukeAndReturn()
+    }
     if (
       keys.filter(
-        key => !['locks', 'account', 'balance', 'network'].includes(key)
+        key =>
```

Comment:
Finding out that this was an issue was very difficult. I'm going to file an issue about the silent failures in the paywall (of which there are at least 2, but probably more)